build/common.targets

```diff
@@ -11,6 +11,11 @@
     <IsCore>true</IsCore>
   </PropertyGroup>
 
+  <PropertyGroup Condition=" '$(TargetFramework)' == 'netcoreapp3.0' ">
+    <DefineConstants>$(DefineConstants);IS_NETCORE30</DefineConstants>
```

Comment:
Funny thing is there are env vars already defined, but I can't find the pattern :)

Reply:
It's NETCORE3_0. 

You can remove this and replace IS_NETCORE30 with NETCORE3_0 in your cs files.

Reply:
Thanks! Changed accordingly.